server/routes/api/user.ts

```diff
@@ -442,14 +462,14 @@ userRoutes.route("/export").get(isAdmin, async (request, response): Promise<void
 		for (let branchName of await Branches.BranchConfig.getNames()) {
 			// TODO: THIS IS A PRELIMINARY VERSION FOR HACKGT CATALYST
 			// TODO: Change this to { "accepted": true }
-			let attendingUsers: IUserMongoose[] = await User.find({ "applied": true, "applicationBranch": branchName });
+			let attendingUsers = await User.find({ "applied": true, "applicationBranch": branchName });
```

Comment:
uhhh should probably fix those TODOs above

Reply:
@ehsanmasdar @joel99 Do you think I can just get rid of this `/export` endpoint? It's kind of a hidden feature that we haven't used since Catalyst 1 I think

Reply:
Oh, didn't realize that.  We have graphiql and checkin2 can do CSV exports, so I'm guessing we don't need it.

Reply:
Export data? Yeah, we can grab what we need from db if it becomes relevant if this is an issue. I'd prefer to leave and mark as legacy code for now if it's not blocking, though.